### DISCUSSION TITLE
Confusion about the error analysis after deep potential model is obtained

### DISCUSSION THREAD

**Discussion by KeouLee** [deepmodeling/deepmd-kit#3998] [Q&A]
I was reading this paper: "QDπ: A Quantum Deep Potential Interaction Model for Drug Discovery", and one of the figures in the paper confuses me. The comparison between forces for DFT and ML result. How is the negative force in the figure defined?
![images_large_ct2c01172_0001](https://github.com/user-attachments/assets/1620a6f6-fa50-4741-b0e5-4b504a1ec474)

**Comment by KeouLee** [comment]
Do we arbitrarily choose a direction to plot the force instead of calculating L2 norm of the force, thus both the negative and positive value exist?

**Reply by njzjz** [reply]
Using an L2 norm will cause the direction of the force to be lost. For example, these three vectors (1,0,0), (-1,0,0), and (0,1,0) have the same L2 norm. It's suggested to compare the components of the three axes directly.

**Reply by KeouLee** [reply]
Thanks for your reply Dr. Zeng. I think I get the point!
